book/test-utils/envir_hook.qmd

```diff
@@ -21,6 +21,14 @@ knitr::opts_template$set(
   skip_if_testing = list(
     eval = isFALSE(testthat::is_checking() || testthat::is_testing())
   ),
+  skip_test_strict = list(
+    R.options = list(
+      warn = 0L,
+      warnPartialMatchDollar = FALSE,
+      warnPartialMatchArgs = FALSE,
+      warnPartialMatchAttr = FALSE
+    )
+  ),
```

Comment:
I'm not super happy with this but that's the best thing I could came up with.

Ideally, all testing related logic (here: exclusions) should live inside `test/` directory. In TLG-Catalog, our test cases are actually living inside the `book/` that I would like to avoid if possible. I was struggling with how to exclude from strict testing. Usually I will modify individual test case. Here I needed to modify the articles.
An alternative that I also considered would be to change rendering the book into rendering individual files in a for-loop and try to control exclusions from there. This however, would allow us to control on a file level but we want to control on a lower, individual code chunk level. Therefore I stick with the chunk template and use it where needed.